### PR TITLE
[#4149] fixed modal on iphonex

### LINKED ISSUE #4149: Issue signing Dapp Messages
[comment]: # (Please replace ... with your information. Remove < and >)

### User Story

As a user, I want to sign messages so that I can use Dapps.

### Description

[comment]: # (Feature or Bug? i.e Type: Bug)
*Type*: Bug

[comment]: # (Describe the feature you would like, or briefly summarise the bug and what you did, what you expected to happen, and what actually happens. Sections below)
*Summary*: There is an error when trying to use dapps and sign messages. When tapping `sign` or `cancel` there is a spinner then the Dapp/page refreshes.

#### Expected behavior

Can sign Dapp messages

#### Actual behavior

A spinner is shown and the page refreshes.

### Reproduction

- In Status go to https://status-im.github.io/dapp/
- Tap Transactions
- Tap sign message
- Sign the message. 
- A spinner should be shown before reloading the `accounts` page

signing-refresh

### Additional Information

iOS
Nightly
Iphone X

<blockquote><div><strong><a href="https://status-im.github.io/dapp/">DAPP</a></strong></div></blockquote>

### PR DESCRIPTION
**Pull request by flexsurfer**
fixes #4149

### CODE REVIEW

**Review by goranjovic — APPROVED** [review]

File: src/status_im/ui/screens/views.cljs [hunk 1]
```diff
@@ -113,54 +113,79 @@
...
-                        :network-details network-details
-                        :offline-messaging-settings offline-messaging-settings
-                        :currency-settings currency-settings
-                        :recent-recipients recent-recipients
-                        :recipient-qr-code recipient-qr-code
-                        :contact-code contact-code
-                        :profile-qr-viewer profile.user/qr-viewer
-                        :backup-seed backup-seed
-                        [react/view [react/text (str "Unknown view: " view-id)]])
+      (let [component (get-main-component view-id)
```
Comment (jeluard) [inline src/status_im/ui/screens/views.cljs]: Align values? :)

**Review by jeluard — COMMENTED** [review]
Can you detail (and add comments) about how you fixed the issue?

File: src/status_im/ui/screens/views.cljs (hunk 1 above)
Comment (flexsurfer) [reply src/status_im/ui/screens/views.cljs]: ?

**Comment by flexsurfer** [comment]
@jeluard i just moved modal in a separate view, in that case when modal is changing the main view is not recalculating

File: src/status_im/ui/screens/views.cljs (hunk 1 above)
Comment (rasom) [reply src/status_im/ui/screens/views.cljs]: https://github.com/bbatsov/clojure-style-guide#vertically-align-let-and-map
```clojure
;; good
(let [thing1     "some stuff"
      thing23456 "other stuff"]
  ...)

;; bad
(let [thing1 "some stuff"
      thing23456 "other stuff"]
  ...)
```
but unfortunately cljfmt doesn't detect such things

Comment (rasom) [reply src/status_im/ui/screens/views.cljs]: In Cursive you can configure this at Editor > Code Style > Clojure, General tab, "Align let-binding values" checkbox

Comment (flexsurfer) [reply src/status_im/ui/screens/views.cljs]: thanks, fixed

**Comment by flexsurfer** [comment]
@jeluard fixed

**Review by rasom — APPROVED** [review]

**Comment by Serhy** [comment]
That works great for me on iPhoneX. Both #4149 and #3342 are not reproduced for me.

**Pull request merged**